controllers:
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 100
  wheel_legged_controller:
    type: wheel_legged_controller/WheelLeggedController
  wheel_legged_cheater_controller:
    type: wheel_legged_controller/WheelLeggedCheaterController
